requirements-test.txt

```diff
@@ -5,7 +5,11 @@
 # Kali Linux, so it makes sense to force the installation of Ansible
 # 2.10 or newer.
 ansible>=2.10,<3
-ansible-lint
+# ansible-lint has a number of breaking changes with the update to v5 as we have
+# documented in https://github.com/cisagov/skeleton-ansible-role/issues/69
+# We can unpin once everything with the v5 update has shaken out and we have a
+# clear path forward.
+ansible-lint<5
 boto3
```

Comment:
Is `boto3` needed here?

Reply:
Not anymore, removed in 2ea865d.

Reply:
Just kidding. It's needed because a dependency uses an SSM lookup. I dropped the commit. Shall we add a comment explaining why it's needed?

Reply:
Yes, please add a comment for that.

Reply:
Comment added in 1aa4b11. Sorry for the confusion.